src/xopen/__init__.py

```diff
@@ -96,7 +96,7 @@
     _MAX_PIPE_SIZE = None
 
 
-FilePath = Union[str, bytes, os.PathLike]
+FilePath = Union[str, bytes, os.PathLike, IO]
```

Comment:
This should be renamed to reflect the change in meaning. It should probably be `FileOrPath`.